eq-author/src/components/Comments/Comment/index.js

```diff
@@ -0,0 +1,225 @@
+import React, { useState } from "react";
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+import CommentEditor from "../CommentEditor";
+import Collapsible from "components/Collapsible";
+import Button from "components/buttons/Button";
+
+import PureComment from "../PureComment";
+
+const StyledCollapsible = styled(Collapsible)`
+  .collapsible-title,
+  .collapsible-title > * {
+    font-weight: normal;
+    padding: 0;
+  }
+
+  .collapsible-header {
+    margin-bottom: 1em;
+  }
+
+  .collapsible-body {
+    border-left: none;
+    margin-top: 0;
+  }
+
+  ul {
+    padding: 0;
+
+    li {
+      display: block;
+    }
+  }
+`;
+
+const Replies = ({
+  replies,
+  commentId,
+  onUpdateReply,
+  onDeleteReply,
+  showAddReply,
+  showReplyBtn,
+}) => {
+  const numOfReplies = replies.length;
+
+  return (
+    <>
+      {numOfReplies === 0 && <React.Fragment />}
```

Comment:
not sure what this is doing

Reply:
If there's no replies, it chucks out a fragment instead to not display the replies toggle. But, I could instead swap it to just return nothing from the component if there's no replies. I'll take a look at this.